Documentation/security/policy/language.rst

```diff
@@ -753,6 +760,29 @@ only be able to emit packets using TCP on port 80, to any layer 3 destination:
 
         .. literalinclude:: ../../../examples/policies/l4/l4.json
 
+Example Port Ranges
+~~~~~~~~~~~~~~~~~~~
+
+The following rule limits all endpoints with the label ``app=myService`` to
+only be able to emit packets using TCP on ports 80-444, to any layer 3 destination:
+
+.. only:: html
+
+   .. tabs::
+     .. group-tab:: k8s YAML
+
+        .. literalinclude:: ../../../examples/policies/l4/l4_port_range.yaml
+     .. group-tab:: JSON
+
+        .. literalinclude:: ../../../examples/policies/l4/l4_port_range.json
+
+.. only:: epub or latex
+
+        .. literalinclude:: ../../../examples/policies/l4/l4_port_range.json
+
+.. note::  "endPort" is ignored when when "port" is a named port.
```

Comment:
A few points here:
- We already declared this above. Do we need to write it twice?
- In context, this is written as a callout immediately after an unrelated example. Should it be elsewhere?
- Does this need to be a callout? I would argue that unless there's something particularly notable about this, it should just be plain text and not inside a `.. note::` stanza. For instance if it will catch out a significant number of people unless we call it out then it may make sense. I doubt that named ports are so frequently used or are likely to be used in conjunction with port ranges.
```suggestion
```